Mathlib/MeasureTheory/Order/Group/Lattice.lean

```diff
@@ -16,11 +16,16 @@ import Mathlib.MeasureTheory.Order.Lattice
 measurable function, group, lattice operation
 -/
 
-variable {R : Type*} [Lattice R] [Group R] [MeasurableSpace R] [MeasurableSup₂ R]
+variable {R S : Type*} [Lattice R] [Group R] [MeasurableSpace R] [MeasurableSup₂ R]
+  [Lattice S] [Group S] [MeasurableSpace S] {f : S → R} (hf : Measurable f)
```

Comment:
Your algebra brain is making you see rings everywhere 😀
```suggestion
variable {α β : Type*} [Lattice α] [Group β] [MeasurableSpace α] [MeasurableSup₂ α]
  [Lattice β] [Group β] [MeasurableSpace β] {f : β → α} (hf : Measurable f)
```

Reply:
You don't need Lattice or Group structures on `S`/`β`, do you?

Reply:
That too

Reply:
> Your algebra brain is making you see rings everywhere 😀

Indeed. Still `α` needs to be a group so maybe I should call it `G`?

I made the changes. I think we're getting there.